app/main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str, skills: list = None) -> None:
```

Comment:
![image](https://user-images.githubusercontent.com/58142395/228867262-68882963-3c39-4dd9-b304-c97c45ffd556.png)
this class take only one parameters